torchvision/transforms/transforms.py

```diff
@@ -687,3 +687,70 @@ def __call__(self, img):
         if random.random() < self.p:
             return F.to_grayscale(img, num_output_channels=num_output_channels)
         return img
+
+
+class RandomErasing(object):
+    """ Randomly selects a rectangle region in an image and erases its pixels with random values,
+        or the Imagenet mean pixel value.
+        'Random Erasing Data Augmentation' by Zhong et al.
+        See https://arxiv.org/pdf/1708.04896.pdf
+    Args:
+         probability: The probability that the Random Erasing operation will be performed.
+         sl: Minimum proportion of erased area against input image.
+         sh: Maximum proportion of erased area against input image.
+         r1: Minimum aspect ratio of erased area.
+         values: Type of erasing value.
+                 If values = 0, erasing with random values,
+                 else, erasing with the Imagenet mean pixel value.
+    Returns:
+        Erased Image.
+    Examples:
+        >>> transform = transforms.Compose([
+        >>> transforms.RandomHorizontalFlip(),
+        >>> transforms.ToTensor(),
+        >>> transforms.Normalize((0.485, 0.456, 0.406), (0.229, 0.224, 0.225)),
+        >>> transforms.RandomErasing(),
+        >>> ])
+    """
+
+    def __init__(self, probability=0.5, sl=0.02, sh=0.4, r1=0.3, value=0):
```

Comment:
As I said before, it might be better to let `value` be a general Tensor or number, that can be broadcast to the correct shape

Reply:
@fmassa Thanks for your advice, I have updated the code. The Random Erasing is efficient in the classification task. We achieve reasonable improvements on CIFAR-10, CIFAR-100, and Fashion-MNISt datasets with various CNN models. In further, Random Erasing can also improve the baselines in object detection and person re-identification tasks. Therefore, I believe it will be widely used in CNN training for most user.